tests/Feature/Api/Place/PlaceControllerTest.php

```diff
@@ -235,7 +236,7 @@ public function testGetPlaceCollectionSearchWithoutFilters()
 
         $response =  $this->actingWithToken()->json(
             'GET',
-            "/api/v1/places/search?filter[category]=&filter[location]=&page=1"
+            "/api/v1/places/search?filter[category]=&filter[location]=&filter[polygon]=&page=1"
```

Comment:
testGetPlaceCollectionSearchWithoutFilters() and `"/api/v1/places/search?filter[category]=&filter[location]=&filter[polygon]=&page=1"` is incompatible, but not required to fix